test/graaflib/algorithm/minimum_spanning_tree_test.cpp

```diff
@@ -0,0 +1,70 @@
+#include <cassert>
+#include <iostream>
+
+#include "minimum_spanning_tree.h"
```

Comment:
```suggestion

#include <graaflib/algorithm/minimum_spanning_tree.h>
```

Reply:
#include <graaflib/algorithm/minimum_spanning_tree.h>

Getting this error again : - graaflib/algorithm/minimum_spanning_tree.h: No such file or directory